.chloggen/spanmetricsprocessor-emit-every-n-seconds.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
The aim was to avoid a breaking change on _what_ metrics are emitted by publishing all metrics since startup. However, _how_ those metrics are emitted is a breaking change, in that metrics are emitted every n seconds instead of immediately after receiving the span batch. As such, I've changed this to `breaking` in [d56c30d](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16601/commits/d56c30d3e36ee86681eabec93f027840904a33ba).